mapserver/django/django_project/mapserver/tasks.py

```diff
@@ -0,0 +1,12 @@
+import logging
+from celery import shared_task
+from django.core import management
+
+logger = logging.getLogger(__name__)
+
+
+@shared_task(name='mapserver.process_hazard_event_queue')
```

Comment:
not associated with mapserver. This is just a generic task. Let's use `fba` namespace.